frontend/src/utils/format.ts

```diff
@@ -72,7 +72,12 @@ export function getShipmentStatusBadgeColor(
  * @example "UK-2021-03"
  */
 export function formatShipmentName(
-  shipment: Shipment | ShipmentQuery['shipment'],
+  shipment:
+    | Shipment
+    | Pick<
+        ShipmentQuery['shipment'],
+        'labelMonth' | 'labelYear' | 'shippingRoute'
+      >,
```

Comment:
This allows us to use this method when the `shipment` object only has a subset of the fields.